src/pages/program/[slug]/[profileRoute]/settings.tsx

```diff
@@ -96,13 +96,13 @@ const SettingsPage: Page = () => {
       <div className="h-4"></div>
       <Card className="flex flex-col p-12 space-y-6 overflow-y-auto">
         <Text h3 b>
-          Mentorship Details
+          Program Details
         </Text>
         <Text className="text-error">{err}</Text>
 
         <div className="grid grid-cols-4 gap-4 justify-center items-center">
           <Text className="col-span-1" b secondary>
-            Mentorship Icon:
+            Program Icon:
```

Comment:
Noah said that he wanted to change it so that organizations don't think that this has to be some kind of formal mentorship. "We want to steer away from having mentor center centered around mentorships"

Reply:
Gotcha--sounds good! (hmmm "Mentor Center" might needa rebrand or smth then lololol)